editor/editor_help.cpp

```diff
@@ -1389,8 +1399,17 @@ void EditorHelp::_update_doc() {
 			class_desc->add_newline();
 
 			for (KeyValue<String, Vector<DocData::ConstantDoc>> &E : enums) {
-				enum_line[E.key] = class_desc->get_paragraph_count() - 2;
+				String key = E.key;
+				if ((key.get_slice_count(".") > 1) && (key.get_slice(".", 0) == edited_class)) {
+					key = key.get_slice(".", 1);
+				}
+				if (cd.enums.has(key)) {
+					if (cd.enums[key].description.strip_edges().is_empty() && E.key.begins_with("_")) {
```

Comment:
Above the enums are filtered on script docs, but not here, is that an oversight?
```suggestion
					if (cd.is_script_doc && cd.enums[key].description.strip_edges().is_empty() && E.key.begins_with("_")) {
```

Reply:
correct me if I'm wrong but we shouldn't have to check cd.is_script_doc every time because it doesn't change per enum. 
the reason we are checking if the description is empty or if the name starts with "_" is it is possible for enums that don't need documentation generated to be in the for loop.

Reply:
Then there should be some other flag to control this as the filtering is on script docs only,

Reply:
oh I didn't realize we only filter on script docs.